### PR TITLE
fix: make clean up pick up all org projects

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

clean up is not picking up all projects

### CODE REVIEW

File: test/e2e/atlas/alerts_test.go [hunk 1]
```diff
@@ -85,6 +83,9 @@ func TestAlerts(t *testing.T) {
 	})
 
 	t.Run("Describe", func(t *testing.T) {
+		if alertID == "" {
+			t.Skip("no alert found")
```
Comment (gssbzn) [inline test/e2e/atlas/alerts_test.go:87]: make it tolerant to not finding an alert config

File: test/e2e/atlas/cleanup_test.go [hunk 2]
```diff
@@ -41,67 +41,62 @@ func TestCleanup(t *testing.T) {
...
-		projectEntity,
+	args := []string{projectEntity,
 		"list",
 		"--limit=500",
-		"-o=json")
+		"-o=json",
+	}
+	if orgID, set := os.LookupEnv("MCLI_ORG_ID"); set {
+		args = append(args, "--orgId", orgID)
+	}
```
Comment (gssbzn) [inline test/e2e/atlas/cleanup_test.go:51]: fix to always list org projects

File: test/e2e/atlas/cleanup_test.go [hunk 3]
```diff
@@ -41,67 +41,62 @@ func TestCleanup(t *testing.T) {
...
 		}
-		func(projectId string) {
-			t.Run(fmt.Sprintf("trying to delete project %s", projectId), func(t *testing.T) {
-				t.Parallel()
-				t.Run(fmt.Sprintf("trying to delete project's %s resources", projectId), func(t *testing.T) {
+		t.Run(fmt.Sprintf("trying to delete project %s", projectID), func(t *testing.T) {
+			t.Parallel()
+			t.Cleanup(func() {
+				deleteProjectWithRetry(t, projectID)
+			})
```
Comment (gssbzn) [inline test/e2e/atlas/cleanup_test.go:69]: move closer to the top and avoid overlooking this call

File: test/e2e/atlas/cleanup_test.go [hunk 4]
```diff
@@ -41,67 +41,62 @@ func TestCleanup(t *testing.T) {
...
+			for _, provider := range []string{"aws", "gcp", "azure"} {
+				p := provider
+				t.Run(fmt.Sprintf("delete network peers for %s", p), func(t *testing.T) {
 					t.Parallel()
-					for _, provider := range []string{"aws", "gcp", "azure"} {
-						func(provider string) {
```
Comment (gssbzn) [inline test/e2e/atlas/cleanup_test.go:70]: same, no need for an iife

**Review by wtrocki — APPROVED** [review]
LGTM.

**Comment by wtrocki** [comment]
Terraform cleanup could consider reusing that code so we have one place to maintain the general cleanup.

**Comment by colm-quinn** [comment]
> Terraform cleanup could consider reusing that code so we have one place to maintain the general cleanup.

cc @marcosuma

**Pull request merged**